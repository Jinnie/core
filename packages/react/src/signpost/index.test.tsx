import * as React from 'react';
import { mount, shallow } from 'enzyme';
import { CdsSignpost } from './index';

describe('CdsSignpost', () => {
  it('renders', () => {
    const wrapper = shallow(
      <div>
        <CdsSignpost></CdsSignpost>
      </div>
    );
    const renderedComponent = wrapper.find(CdsSignpost);
    expect(renderedComponent).toBeDefined();
  });

  it('snapshot', () => {
    const vertDivStyle = {
      height: '140px',
      marginTop: '24px',
    };
    const wrapper = mount(
      <div>
        <CdsSignpost></CdsSignpost>
        <div style={vertDivStyle}>
          <CdsSignpost orientation="vertical">1</CdsSignpost>
        </div>
      </div>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
