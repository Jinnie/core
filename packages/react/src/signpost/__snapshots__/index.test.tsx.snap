// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CdsSignpost snapshot 1`] = `
<div>
  <CdsSignpost>
    <CdsSignpost
      l={null}
    >
      <cds-signpost
        l={null}
      />
    </CdsSignpost>
  </CdsSignpost>
  <div
    style={
      Object {
        "height": "140px",
        "marginTop": "24px",
      }
    }
  >
    <CdsSignpost
      orientation="vertical"
    >
      <CdsSignpost
        l={null}
        orientation="vertical"
      >
        <cds-signpost
          l={null}
        >
          1
        </cds-signpost>
      </CdsSignpost>
    </CdsSignpost>
  </div>
</div>
`;
